### ISSUE TITLE
Whats the value to set environment variable EXECJS_RUNTIME?

### ISSUE THREAD

**Issue by Padmahas** [devinus/execjs#5]
Hello Devin Torres, [here](https://github.com/devinus/execjs) you have said "EXECJS_RUNTIME environment variable can also be used to set the runtime". I used `export  EXECJS_RUNTIME="/opt/node-v4.4.5-linux-x64/bin"` but when I run _rails s_, it shows `Gem Load Error is: wrong constant name /opt/node-v4.4.5-linux-x64/bin`.

So what must be the value of EXECJS_RUNTIME?
Thank you.

**Comment by devinus** [comment]
@Padmahas are you looking for the Ruby `execjs`? https://github.com/rails/execjs

**Comment by Padmahas** [comment]
No. But its runtime. I would like to install softwares manually. So I installed .tar.xz version NodeJS in my Ubuntu system inside /opt/nodejs... but rails can't find it (may be its searching in default location). So after some digging I came to know that setting EXECJS_RUNTIME environment variable is one of the best solution. While googling I came across your instructions to set env var. So what must be the value for that variable? As I mentioned in the previous comment is it the **bin** directory of **NodeJS** or what? What must be the value of EXECJS_RUNTIME?
